packages/web/src/styles/Input.js

```diff
@@ -7,13 +7,13 @@ const alertBorder = ({ theme }) => css`
 
 const input = css`
 	width: 100%;
-	height: 42px;
-	line-height: 42px;
+	line-height:1.5;
```

Comment:
@mohdashraf010897 You've removed the `height` as `42px` which breaks the default UI of `DataSearch` component. Same issue is with vue.